CHANGELOG.md

```diff
@@ -419,98 +517,3 @@ Infrastructure and code quality improvements
 
 ---
 
-# v0.1.2
```

Comment:
Is it intentional that these releases have been deleted?

Reply:
Somehow the python script deleted some of the content even though the tags still exist. Let me revert this change manually